packages/services/f-consumer-oidc/webpack.config.js

```diff
@@ -1,37 +0,0 @@
-const path = require('path');
-const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
-
-module.exports = [
-    'source-map'
-].map(devtool => ({
-    mode: 'development',
-    entry: './src/index.js',
-    module: {
-        rules: [
-            {
-                test: /\.js$/,
-                exclude: /node_modules/,
-                use: {
-                    loader: 'babel-loader'
-                }
-            }
-        ]
-    },
-    output: {
-        path: path.resolve(__dirname, 'dist'),
-        filename: 'f-consumer-oidc.umd.min.js',
-        library: 'f-consumer-oidc',
-        libraryTarget: 'umd',
-        globalObject: 'this'
-    },
-    optimization: {
-        minimize: true,
-        minimizer: [new UglifyJsPlugin({
-            include: /\.min\.js$/
-        })]
-    },
-    externals: {
-        'js-cookie': 'js-cookie'
```

Comment:
Do you need to do anything in Vite to handle peer deps?

Reply:
Ah that's a good catch! I'll check. Thanks

Reply:
Should be fixed now 👍